src/JsonLd/Action/ContextAction.php

```diff
@@ -48,22 +49,46 @@ public function __construct(ContextBuilderInterface $contextBuilder, ResourceNam
      */
     public function __invoke(string $shortName): array
     {
-        if ('Entrypoint' === $shortName) {
+        $aPath = 1 === \count(explode('/', $shortName))
```

Comment:
All this logic looks too complicated. IMO we should just use the router to generate the appropriate URL somewhere. I'll take a deeper look ASAP.